<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    
    <!-- Website verification -->
    <meta name="google-site-verification" content="lGO5GJnWH-wiyh99r6x1XHoaVf3HOHJGHmdyQ5GR37E">
<!-- Avoid warning on Google Chrome
        Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.
        see https://stackoverflow.com/a/75119417
    -->
    <meta http-equiv="Permissions-Policy" content="interest-cohort=()">

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Publications | Yunzhe  Jiang</title>
    <meta name="author" content="Yunzhe  Jiang">
    <meta name="description" content="Yunzhe Jiang, a Ph.D. student in Computational Biology and Bioinformatics at Yale.
">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Bootstrap Table -->
    <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/favicon.png">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://yzjiang9.github.io/publications/">

    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Yunzhe </span>Jiang</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">About</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/presentations/">Presentations</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/cv/">CV</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/teaching/">Teaching</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Publications</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <div class="publications">

<h1>Preprints</h1>

  <h2 class="year">2025</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
        </div>

        <!-- Entry bib key -->
        <div id="Chothani2025.07.03.662928" class="col-sm-8">
        <!-- Title -->
        <div class="title">An expanded reference catalog of translated open reading frames for biomedical research</div>
        <!-- Author -->
        <div class="author">
        

        Sonia Chothani, Jorge Ruiz-Orera, Jack A. S. Tierney, Jim Clauwaert, Eric W Deutsch, M. Mar Alba, Julie L. Aspden, Pavel V. Baranov, Ariel Alejandro Bazzini, Elspeth A. Bruford, Marie A. Brunet, Tristan Cardon, Anne-Ruxandra Carvunis, Claudio Casola, Jyoti Sharma Choudhary, Kellie Dean, Pouya Faridi, Ivo Fierro-Monti, Isabelle Fournier, Adam Frankish, Mark Gerstein, Norbert Hubner, <em>Yunzhe Jiang</em>, Manolis Kellis, Leron W. Kok, Thomas F Martinez, Gerben Menschaert, Pengyu Ni, Sandra Orchard, Xavier Roucou, Joel Rozowsky, Michel Salzet, Mauro Siragusa, Sarah Slavoff, Michal I Swirski, Nicola Ternette, Eivind Valen, Juan Antonio Vizcaino, Aaron Wacholder, Wei Wu, Zhi Xie, Yucheng T. Yang, Robert L. Moritz, Jonathan Mudge, Sebastiaan Heesch, John R Prensner, and Owen JL Rackham</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>bioRxiv</em>, 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Non-canonical (i.e., unannotated) open reading frames (ncORFs) have until recently been omitted from reference genome annotations, despite evidence of their translation, limiting their incorporation into biomedical research. To address this, in 2022, we initiated the TransCODE consortium and built the first community-driven consensus catalog of human ncORFs, which was openly distributed to the research community via Ensembl-GENCODE. While this catalog represented a starting point for reference ncORF annotation, major technical and scientific issues remained. In particular, this initial catalogue had no standardized framework to judge the evidence of translation for individual ncORFs. Here, we present an expanded and refined catalog of the human reference annotation of ncORFs. By incorporating more datasets and by lifting constraints on ORF length and start-codon, we define a comprehensive set of 28,359 ncORFs that is nearly four times the size of the previous catalog. Furthermore, to aid users who wish to work with ncORFs with the strongest and most reproducible signals of translation, we utilized a data-driven framework (i.e. translation signature scores) to assess the accumulated evidence for any individual ncORF. Using this approach, we derive a subset of 7,888 ncORFs with translation evidence on par with canonical protein-coding genes, which we refer to as the Primary set. This set can serve as a reliable reference for downstream analyses and validation, with a particular emphasis on high quality. Overall, this update reflects continual community-driven efforts to make ncORFs accessible and actionable to the broader research public and further iterations of the catalog will continue to expand and refine this resource.Competing Interest StatementJ.R.P. has received research honoraria from Novartis Biosciences and Quantum-Si, and is a paid consultant for ProFound Therapeutics. J.L.A. is an advisor to Microneedle Solutions. G.M. is co-founder and CSO of OHMX.bio. P.F. is a member of the scientific advisory board of Infinitopes. A.-R. C. is a member of the advisory board of ProFound Therapeutics. P.V.B. is a cofounder and shareholder of Eirnabio Ltd.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
        </div>

        <!-- Entry bib key -->
        <div id="Yang2025.02.05.636707" class="col-sm-8">
        <!-- Title -->
        <div class="title">Multi-omic Characterization of HIV Effects at Single Cell Level across Human Brain Regions</div>
        <!-- Author -->
        <div class="author">
        

        Junchen Yang, Kriti Agrawal, Jay Stanley, Ruiqi Li, Nicholas Jacobs, Haowei Wang, Chang Lu, Rihao Qu, Declan Clarke, Yuhang Chen, <em>Yunzhe Jiang</em>, Donglu Bai, Suchen Zheng, Howard Fox, Ya-chi Ho, Anita Huttner, Mark Gerstein, Yuval Kluger, Le Zhang, and Serena Spudich</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>bioRxiv</em>, 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>HIV infection exerts profound and long-lasting neurodegenerative effects on the central nervous system (CNS) that can persist despite antiretroviral therapy (ART). Here, we used single-nucleus multiome sequencing to map the transcriptomic and epigenetic landscapes of postmortem human brains from 13 healthy individuals and 20 individuals with HIV who have a history of treatment with ART. Our study spanned three distinct regions—the prefrontal cortex, insular cortex, and ventral striatum—enabling a comprehensive exploration of region-specific and cross-regional perturbations. We found widespread and persistent HIV-associated transcriptional and epigenetic alterations across multiple cell types. Detailed analyses of microglia revealed state changes marked by immune activation and metabolic dysregulation, while integrative multiomic profiling of astrocytes identified multiple subpopulations, including a reactive subpopulation unique to HIV-infected brains. These findings suggest that cells from people with HIV exhibit molecular shifts that may underlie ongoing neuroinflammation and CNS dysfunction. Furthermore, cell–cell communication analyses uncovered dysregulated and pro-inflammatory interactions among glial populations, underscoring the multifaceted and enduring impact of HIV on the brain milieu. Collectively, our comprehensive atlas of HIV-associated brain changes reveals distinct glial cell states with signatures of pro-inflammatory signaling and metabolic dysregulation, providing a framework for developing targeted therapies for HIV-associated neurological dysfunction.Competing Interest StatementThe authors have declared no competing interest.</p>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2024</h2>
  <ol class="bibliography"></ol>

  <h2 class="year">2023</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/encode4.webp">
          
        </div>

        <!-- Entry bib key -->
        <div id="Reese2023.05.15.540865" class="col-sm-8">
        <!-- Title -->
        <div class="title">The ENCODE4 long-read RNA-seq collection reveals distinct classes of transcript structure diversity</div>
        <!-- Author -->
        <div class="author">
        

        Fairlie Reese, Brian Williams, Gabriela Balderrama-Gutierrez, Dana Wyman, Muhammed Hasan Çelik, Elisabeth Rebboah, Narges Rezaie, Diane Trout, Milad Razavi-Mohseni, <em>Yunzhe Jiang</em>, Beatrice Borsari, Samuel Morabito, Heidi Yahan Liang, Cassandra J. McGill, Sorena Rahmanian, Jasmine Sakr, Shan Jiang, Weihua Zeng, Klebea Carvalho, Annika K. Weimer, Louise A. Dionne, Ariel McShane, Karan Bedi, Shaimae I. Elhajjajy, Sean Upchurch, Jennifer Jou, Ingrid Youngworth, Idan Gabdank, Paul Sud, Otto Jolanki, J. Seth Strattan, Meenakshi S. Kagda, Michael P. Snyder, Ben C. Hitz, Jill E. Moore, Zhiping Weng, David Bennett, Laura Reinholdt, Mats Ljungman, Michael A. Beer, Mark B. Gerstein, Lior Pachter, Roderic Guigó, Barbara J. Wold, and Ali Mortazavi</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>bioRxiv</em>, 2023
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The majority of mammalian genes encode multiple transcript isoforms that result from differential promoter use, changes in exonic splicing, and alternative 3’ end choice. Detecting and quantifying transcript isoforms across tissues, cell types, and species has been extremely challenging because transcripts are much longer than the short reads normally used for RNA-seq. By contrast, long-read RNA-seq (LR-RNA-seq) gives the complete structure of most transcripts. We sequenced 264 LR-RNA-seq PacBio libraries totaling over 1 billion circular consensus reads (CCS) for 81 unique human and mouse samples. We detect at least one full-length transcript from 87.7% of annotated human protein coding genes and a total of 200,000 full-length transcripts, 40% of which have novel exon junction chains.To capture and compute on the three sources of transcript structure diversity, we introduce a gene and transcript annotation framework that uses triplets representing the transcript start site, exon junction chain, and transcript end site of each transcript. Using triplets in a simplex representation demonstrates how promoter selection, splice pattern, and 3’ processing are deployed across human tissues, with nearly half of multitranscript protein coding genes showing a clear bias toward one of the three diversity mechanisms. Evaluated across samples, the predominantly expressed transcript changes for 74% of protein coding genes. In evolution, the human and mouse transcriptomes are globally similar in types of transcript structure diversity, yet among individual orthologous gene pairs, more than half (57.8%) show substantial differences in mechanism of diversification in matching tissues. This initial large-scale survey of human and mouse long-read transcriptomes provides a foundation for further analyses of alternative transcript usage, and is complemented by short-read and microRNA data on the same samples and by epigenome data elsewhere in the ENCODE4 collection.Competing Interest StatementThe authors have declared no competing interest.</p>
          </div>
        </div>
      </div>
</li></ol>


<h1>Peer-reviewed Paper</h1>

  <h2 class="year">2025</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
        </div>

        <!-- Entry bib key -->
        <div id="coorens_human_2025" class="col-sm-8">
        <!-- Title -->
        <div class="title">The human and non-human primate developmental GTEx projects</div>
        <!-- Author -->
        <div class="author">
        

        Tim H. H. Coorens, Amy Guillaumet-Adkins, Rothem Kovner, Rebecca L. Linn, Victoria H. J. Roberts, Amrita Sule, Patrick M. Van Hoose, and  Consortium</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature</em>, Jan 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1038/s41586-024-08244-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://doi.org/10.1038/s41586-024-08244-9.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Many human diseases are the result of early developmental defects. As most paediatric diseases and disorders are rare, children are critically underrepresented in research. Functional genomics studies primarily rely on adult tissues and lack critical cell states in specific developmental windows. In parallel, little is known about the conservation of developmental programmes across non-human primate (NHP) species, with implications for human evolution. Here we introduce the developmental Genotype-Tissue Expression (dGTEx) projects, which span humans and NHPs and aim to integrate gene expression, regulation and genetics data across development and species. The dGTEx cohort will consist of 74 tissue sites across 120 human donors from birth to adulthood, and developmentally matched NHP age groups, with additional prenatal and adult animals, with 126 rhesus macaques (Macaca mulatta) and 72 common marmosets (Callithrix jacchus). The data will comprise whole-genome sequencing, extensive bulk, single-cell and spatial gene expression profiles, and chromatin accessibility data across tissues and development. Through community engagement and donor diversity, the human dGTEx study seeks to address disparities in genomic research. Thus, dGTEx will provide a reference human and NHP dataset and tissue bank, enabling research into developmental changes in expression and gene regulation, childhood disorders and the effect of genetic variation on development.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/hsgv.webp">
          
        </div>

        <!-- Entry bib key -->
        <div id="logsdon_complex_2025" class="col-sm-8">
        <!-- Title -->
        <div class="title">Complex genetic variation in nearly complete human genomes</div>
        <!-- Author -->
        <div class="author">
        

        Glennis A. Logsdon, Peter Ebert, Peter A. Audano, Mark Loftus, David Porubsky, Jana Ebler, Feyza Yilmaz, Pille Hallast, Timofey Prodanov, DongAhn Yoo, Carolyn A. Paisie, William T. Harvey, Xuefang Zhao, Gianni V. Martino, Mir Henglin, Katherine M. Munson, Keon Rabbani, Chen-Shan Chin, Bida Gu, Hufsah Ashraf, Stephan Scholz, Olanrewaju Austine-Orimoloye, Parithi Balachandran, Marc Jan Bonder, Haoyu Cheng, Zechen Chong, Jonathan Crabtree, Mark Gerstein, Lisbeth A. Guethlein, Patrick Hasenfeld, Glenn Hickey, Kendra Hoekzema, Sarah E. Hunt, Matthew Jensen, <em>Yunzhe Jiang</em>, Sergey Koren, Youngjun Kwon, Chong Li, Heng Li, Jiaqi Li, Paul J. Norman, Keisuke K. Oshima, Benedict Paten, Adam M. Phillippy, Nicholas R. Pollock, Tobias Rausch, Mikko Rautiainen, Yuwei Song, Arda Söylev, Arvis Sulovari, Likhitha Surapaneni, Vasiliki Tsapalou, Weichen Zhou, Ying Zhou, Qihui Zhu, Michael C. Zody, Ryan E. Mills, Scott E. Devine, Xinghua Shi, Michael E. Talkowski, Mark J. P. Chaisson, Alexander T. Dilthey, Miriam K. Konkel, Jan O. Korbel, Charles Lee, Christine R. Beck, Evan E. Eichler, and Tobias Marschall</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature</em>, Aug 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-025-09140-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://www.nature.com/articles/s41586-025-09140-6.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Diverse sets of complete human genomes are required to construct a pangenome reference and to understand the extent of complex structural variation. Here we sequence 65 diverse human genomes and build 130 haplotype-resolved assemblies (median continuity of 130 Mb), closing 92% of all previous assembly gaps1,2 and reaching telomere-to-telomere status for 39% of the chromosomes. We highlight complete sequence continuity of complex loci, including the major histocompatibility complex (MHC), SMN1/SMN2, NBPF8 and AMY1/AMY2, and fully resolve 1,852 complex structural variants. In addition, we completely assemble and validate 1,246 human centromeres. We find up to 30-fold variation in α-satellite higher-order repeat array length and characterize the pattern of mobile element insertions into α-satellite higher-order repeat arrays. Although most centromeres predict a single site of kinetochore attachment, epigenetic analysis suggests the presence of two hypomethylated regions for 7% of centromeres. Combining our data with the draft pangenome reference1 significantly enhances genotyping accuracy from short-read data, enabling whole-genome inference3 to a median quality value of 45. Using this approach, 26,115 structural variants per individual are detected, substantially increasing the number of structural variants now amenable to downstream disease association studies.</p>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2024</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/gencode2025.jpeg">
          
        </div>

        <!-- Entry bib key -->
        <div id="10.1093/nar/gkae1078" class="col-sm-8">
        <!-- Title -->
        <div class="title">GENCODE 2025: reference gene annotation for human and mouse</div>
        <!-- Author -->
        <div class="author">
        

        Jonathan M Mudge, Sílvia Carbonell-Sala, Mark Diekhans, Jose Gonzalez Martinez, Toby Hunt, Irwin Jungreis, Jane E Loveland, Carme Arnan, If Barnes, Ruth Bennett, Andrew Berry, Alexandra Bignell, Daniel Cerdán-Vélez, Kelly Cochran, Lucas T Cortés, Claire Davidson, Sarah Donaldson, Cagatay Dursun, Reham Fatima, Matthew Hardy, Prajna Hebbar, Zoe Hollis, Benjamin T James, <em>Yunzhe Jiang</em>, Rory Johnson, Gazaldeep Kaur, Mike Kay, Riley J Mangan, Miguel Maquedano, Laura Martínez Gómez, Nourhen Mathlouthi, Ryan Merritt, Pengyu Ni, Emilio Palumbo, Tamara Perteghella, Fernando Pozo, Shriya Raj, Cristina Sisu, Emily Steed, Dulika Sumathipala, Marie-Marthe Suner, Barbara Uszczynska-Ratajczak, Elizabeth Wass, Yucheng T Yang, Dingyao Zhang, Robert D Finn, Mark Gerstein, Roderic Guigó, Tim J P Hubbard, Manolis Kellis, Anshul Kundaje, Benedict Paten, Michael L Tress, Ewan Birney, Fergal J Martin, and Adam Frankish</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nucleic Acids Research</em>, Nov 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1093/nar/gkae1078" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://academic.oup.com/nar/advance-article-pdf/doi/10.1093/nar/gkae1078/60766176/gkae1078.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>GENCODE produces comprehensive reference gene annotation for human and mouse. Entering its twentieth year, the project remains highly active as new technologies and methodologies allow us to catalog the genome at ever-increasing granularity. In particular, long-read transcriptome sequencing enables us to identify large numbers of missing transcripts and to substantially improve existing models, and our long non-coding RNA catalogs have undergone a dramatic expansion and reconfiguration as a result. Meanwhile, we are incorporating data from state-of-the-art proteomics and Ribo-seq experiments to fine-tune our annotation of translated sequences, while further insights into function can be gained from multi-genome alignments that grow richer as more species’ genomes are sequenced. Such methodologies are combined into a fully integrated annotation workflow. However, the increasing complexity of our resources can present usability challenges, and we are resolving these with the creation of filtered genesets such as MANE Select and GENCODE Primary. The next challenge is to propagate annotations throughout multiple human and mouse genomes, as we enter the pangenome era. Our resources are freely available at our web portal www.gencodegenes.org, and via the Ensembl and UCSC genome browsers.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/epact.webp">
          
        </div>

        <!-- Entry bib key -->
        <div id="zhang2024epitope" class="col-sm-8">
        <!-- Title -->
        <div class="title">Epitope-anchored contrastive transfer learning for paired CD8+ T cell receptor–antigen recognition</div>
        <!-- Author -->
        <div class="author">
        

        Yumeng Zhang, Zhikang Wang, <em>Yunzhe Jiang</em>, Dene R Littler, Mark Gerstein, Anthony W Purcell, Jamie Rossjohn, Hong-Yu Ou, and Jiangning Song</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature Machine Intelligence</em>, Oct 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s42256-024-00913-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://www.nature.com/articles/s42256-024-00913-8.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Understanding the mechanisms of T cell antigen recognition that underpin adaptive immune responses is critical for developing vaccines, immunotherapies and treatments against autoimmune diseases. Despite extensive research efforts, accurate prediction of T cell receptor (TCR)–antigen binding pairs remains a great challenge due to the vast diversity and cross-reactivity of TCRs. Here we propose a deep-learning-based framework termed epitope-anchored contrastive transfer learning (EPACT) tailored to paired human CD8+ TCRs. Harnessing the pretrained representations and co-embeddings of peptide–major histocompatibility complex (pMHC) and TCR, EPACT demonstrated generalizability in predicting binding specificity for unseen epitopes and distinct TCR repertoires. Contrastive learning enabled highly precise predictions for immunodominant epitopes and interpretable analysis of epitope-specific T cells. We applied EPACT to SARS-CoV-2-responsive T cells, and the predicted binding strength aligned well with the surge in spike-specific immune responses after vaccination. We further fine-tuned EPACT on structural data to decipher the residue-level interactions involved in TCR–antigen recognition. EPACT was capable of quantifying interchain distance matrices and identifying contact residues, corroborating the presence of TCR cross-reactivity across multiple tumour-associated antigens. Together, EPACT can serve as a useful artificial intelligence approach with important potential in practical applications and contribute towards the development of TCR-based immunotherapies.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/science.adi5199-fa.jpg">
          
        </div>

        <!-- Entry bib key -->
        <div id="doi:10.1126/science.adi5199" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single-cell genomics and regulatory networks for 388 human brains</div>
        <!-- Author -->
        <div class="author">
        

        Prashant S. Emani, Jason J. Liu, Declan Clarke, Matthew Jensen, Jonathan Warrell, Chirag Gupta, Ran Meng, Che Yu Lee, Siwei Xu, Cagatay Dursun, Shaoke Lou, Yuhang Chen, Zhiyuan Chu, Timur Galeev, Ahyeon Hwang, Yunyang Li, Pengyu Ni, Xiao Zhou, PsychENCODE Consortium, Trygve E. Bakken, Jaroslav Bendl, Lucy Bicks, Tanima Chatterjee, Lijun Cheng, Yuyan Cheng, Yi Dai, Ziheng Duan, Mary Flaherty, John F. Fullard, Michael Gancz, Diego Garrido-Martín, Sophia Gaynor-Gillett, Jennifer Grundman, Natalie Hawken, Ella Henry, Gabriel E. Hoffman, Ao Huang, <em>Yunzhe Jiang</em>, Ting Jin, Nikolas L. Jorstad, Riki Kawaguchi, Saniya Khullar, Jianyin Liu, Junhao Liu, Shuang Liu, Shaojie Ma, Michael Margolis, Samantha Mazariegos, Jill Moore, Jennifer R. Moran, Eric Nguyen, Nishigandha Phalke, Milos Pjanic, Henry Pratt, Diana Quintero, Ananya S. Rajagopalan, Tiernon R. Riesenmy, Nicole Shedd, Manman Shi, Megan Spector, Rosemarie Terwilliger, Kyle J. Travaglini, Brie Wamsley, Gaoyuan Wang, Yan Xia, Shaohua Xiao, Andrew C. Yang, Suchen Zheng, Michael J. Gandal, Donghoon Lee, Ed S. Lein, Panos Roussos, Nenad Sestan, Zhiping Weng, Kevin P. White, Hyejung Won, Matthew J. Girgenti, Jing Zhang, Daifeng Wang, Daniel Geschwind, and Mark Gerstein</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Science</em>, Oct 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.science.org/doi/10.1126/science.adi5199" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://www.science.org/doi/epdf/10.1126/science.adi5199" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Single-cell genomics is a powerful tool for studying heterogeneous tissues such as the brain. Yet little is understood about how genetic variants influence cell-level gene expression. Addressing this, we uniformly processed single-nuclei, multiomics datasets into a resource comprising &gt;2.8 million nuclei from the prefrontal cortex across 388 individuals. For 28 cell types, we assessed population-level variation in expression and chromatin across gene families and drug targets. We identified &gt;550,000 cell type–specific regulatory elements and &gt;1.4 million single-cell expression quantitative trait loci, which we used to build cell-type regulatory and cell-to-cell communication networks. These networks manifest cellular changes in aging and neuropsychiatric disorders. We further constructed an integrative model accurately imputing single-cell expression and simulating perturbations; the model prioritized  250 disease-risk genes and drug targets with associated cell types.</p>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2023</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/entex.webp">
          
        </div>

        <!-- Entry bib key -->
        <div id="ROZOWSKY20231493" class="col-sm-8">
        <!-- Title -->
        <div class="title">The EN-TEx resource of multi-tissue personal epigenomes &amp; variant-impact models</div>
        <!-- Author -->
        <div class="author">
        

        Joel Rozowsky, Jiahao Gao, Beatrice Borsari, Yucheng T. Yang, Timur Galeev, Gamze Gürsoy, Charles B. Epstein, Kun Xiong, Jinrui Xu, Tianxiao Li, Jason Liu, Keyang Yu, Ana Berthel, Zhanlin Chen, Fabio Navarro, Maxwell S. Sun, James Wright, Justin Chang, Christopher J.F. Cameron, Noam Shoresh, Elizabeth Gaskell, Jorg Drenkow, Jessika Adrian, Sergey Aganezov, François Aguet, Gabriela Balderrama-Gutierrez, Samridhi Banskota, Guillermo Barreto Corona, Sora Chee, Surya B. Chhetri, Gabriel Conte Cortez Martins, Cassidy Danyko, Carrie A. Davis, Daniel Farid, Nina P. Farrell, Idan Gabdank, Yoel Gofin, David U. Gorkin, Mengting Gu, Vivian Hecht, Benjamin C. Hitz, Robbyn Issner, <em>Yunzhe Jiang</em>, Melanie Kirsche, Xiangmeng Kong, Bonita R. Lam, Shantao Li, Bian Li, Xiqi Li, Khine Zin Lin, Ruibang Luo, Mark Mackiewicz, Ran Meng, Jill E. Moore, Jonathan Mudge, Nicholas Nelson, Chad Nusbaum, Ioann Popov, Henry E. Pratt, Yunjiang Qiu, Srividya Ramakrishnan, Joe Raymond, Leonidas Salichos, Alexandra Scavelli, Jacob M. Schreiber, Fritz J. Sedlazeck, Lei Hoon See, Rachel M. Sherman, Xu Shi, Minyi Shi, Cricket Alicia Sloan, J Seth Strattan, Zhen Tan, Forrest Y. Tanaka, Anna Vlasova, Jun Wang, Jonathan Werner, Brian Williams, Min Xu, Chengfei Yan, Lu Yu, Christopher Zaleski, Jing Zhang, Kristin Ardlie, J Michael Cherry, Eric M. Mendenhall, William S. Noble, Zhiping Weng, Morgan E. Levine, Alexander Dobin, Barbara Wold, Ali Mortazavi, Bing Ren, Jesse Gillis, Richard M. Myers, Michael P. Snyder, Jyoti Choudhary, Aleksandar Milosavljevic, Michael C. Schatz, Bradley E. Bernstein, Roderic Guigó, Thomas R. Gingeras, and Mark Gerstein</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cell</em>, Mar 2023
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(23)00161-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://www.sciencedirect.com/science/article/pii/S0092867423001617/pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Understanding how genetic variants impact molecular phenotypes is a key goal of functional genomics, currently hindered by reliance on a single haploid reference genome. Here, we present the EN-TEx resource of 1,635 open-access datasets from four donors (∼30 tissues × ∼15 assays). The datasets are mapped to matched, diploid genomes with long-read phasing and structural variants, instantiating a catalog of &gt;1 million allele-specific loci. These loci exhibit coordinated activity along haplotypes and are less conserved than corresponding, non-allele-specific ones. Surprisingly, a deep-learning transformer model can predict the allele-specific activity based only on local nucleotide-sequence context, highlighting the importance of transcription-factor-binding motifs particularly sensitive to variants. Furthermore, combining EN-TEx with existing genome annotations reveals strong associations between allele-specific and GWAS loci. It also enables models for transferring known eQTLs to difficult-to-profile tissues (e.g., from skin to heart). Overall, EN-TEx provides rich data and generalizable models for more accurate personal functional genomics.</p>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-3">
          
          <img class="img-fluid" src="/assets/img/gencode.webp">
          
        </div>

        <!-- Entry bib key -->
        <div id="10.1093/nar/gkac1071" class="col-sm-8">
        <!-- Title -->
        <div class="title">GENCODE: reference annotation for the human and mouse genomes in 2023</div>
        <!-- Author -->
        <div class="author">
        

        Adam Frankish, Sílvia Carbonell-Sala, Mark Diekhans, Irwin Jungreis, Jane E Loveland, Jonathan M Mudge, Cristina Sisu, James C Wright, Carme Arnan, If Barnes, Abhimanyu Banerjee, Ruth Bennett, Andrew Berry, Alexandra Bignell, Carles Boix, Ferriol Calvet, Daniel Cerdán-Vélez, Fiona Cunningham, Claire Davidson, Sarah Donaldson, Cagatay Dursun, Reham Fatima, Stefano Giorgetti, Carlos Garcıa Giron, Jose Manuel Gonzalez, Matthew Hardy, Peter W Harrison, Thibaut Hourlier, Zoe Hollis, Toby Hunt, Benjamin James, <em>Yunzhe Jiang</em>, Rory Johnson, Mike Kay, Julien Lagarde, Fergal J Martin, Laura Martínez Gómez, Surag Nair, Pengyu Ni, Fernando Pozo, Vivek Ramalingam, Magali Ruffier, Bianca M Schmitt, Jacob M Schreiber, Emily Steed, Marie-Marthe Suner, Dulika Sumathipala, Irina Sycheva, Barbara Uszczynska-Ratajczak, Elizabeth Wass, Yucheng T Yang, Andrew Yates, Zahoor Zafrulla, Jyoti S Choudhary, Mark Gerstein, Roderic Guigo, Tim J P Hubbard, Manolis Kellis, Anshul Kundaje, Benedict Paten, Michael L Tress, and Paul Flicek</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nucleic Acids Research</em>, Nov 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1093/nar/gkac1071" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://academic.oup.com/nar/article-pdf/51/D1/D942/48440716/gkac1071.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>GENCODE produces high quality gene and transcript annotation for the human and mouse genomes. All GENCODE annotation is supported by experimental data and serves as a reference for genome biology and clinical genomics. The GENCODE consortium generates targeted experimental data, develops bioinformatic tools and carries out analyses that, along with externally produced data and methods, support the identification and annotation of transcript structures and the determination of their function. Here, we present an update on the annotation of human and mouse genes, including developments in the tools, data, analyses and major collaborations which underpin this progress. For example, we report the creation of a set of non-canonical ORFs identified in GENCODE transcripts, the LRGASP collaboration to assess the use of long transcriptomic data to build transcript models, the progress in collaborations with RefSeq and UniProt to increase convergence in the annotation of human and mouse protein-coding genes, the propagation of GENCODE across the human pan-genome and the development of new tools to support annotation of regulatory features by GENCODE. Our annotation is accessible via Ensembl, the UCSC Genome Browser and https://www.gencodegenes.org.</p>
          </div>
        </div>
      </div>
</li></ol>


</div>

          </article>

        </div>

      
    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2025 Yunzhe  Jiang. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Inspired by <a href="https://hrzhang.me/" target="_blanl" rel="external nofollow noopener">Hengrui Zhang</a>.
Last updated: August 20, 2025.
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Bootstrap Table -->
  <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script>

  <!-- Load Common JS -->
  <script src="/assets/js/no_defer.js"></script>
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-YT7MFFPXTR"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){ window.dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'G-YT7MFFPXTR');
  </script>
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
